### PR TITLE
Language setting fixed, code reformatted and optimised

### PR DESCRIPTION
**Pull request by danishjamal104**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/SettingsFragment.kt [hunk 1]
```diff
@@ -16,6 +18,7 @@ import com.mifos.utils.*
 /**
  * Created by mayankjindal on 22/07/17.
  */
+@Suppress("DEPRECATION")
```
Comment (iamsh4shank) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/SettingsFragment.kt:21]: Is this required? as you marked the whole file for deprecation.

**Comment by iamsh4shank** [comment]
Also @danishjamal104 can you check the build fail issue?

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/SettingsFragment.kt (hunk 1 above)
Comment (danishjamal104) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/SettingsFragment.kt]: I have removed this.

**Comment by danishjamal104** [comment]
> Also @danishjamal104 can you check the build fail issue?

I have checked it, its issue with the GitHub workflow with lint task. The build is successful on my local machine checkout below SS.

Screenshot 2021-07-22 at 6 38 20 AM

**Review by iamsh4shank — APPROVED** [review]

**Pull request merged**